plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -689,6 +691,7 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
             my->blocks_dir = app().data_dir() / bld;
          else
             my->blocks_dir = bld;
+         resource_monitor_plugin::monitor_directory(my->blocks_dir);
```

Comment:
`blocks_dir` and `state_dir` need to be added even if not explicitly specified.

Reply:
Agreed.